scripts/st2chatops-startup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# (minimal) check for chatops env
+if [[ -z "$HUBOT_ADAPTER" ]] ; then
+  echo "WARNING: \$HUBOT_ADAPTER is not set, cannot start hubot..."
+  # just sleep indefinitely to avoid this container getting repeatedly restarted
```

Comment:
Having the container running when in fact it had incorrect configuration may bring confusion in terms of liveness and understanding the service health. Failing fast and hard from the misconfigured container would be expected behavior to me.

On top of that, making users configure the restart policy via docker-compose is more preferred:
https://docs.docker.com/compose/compose-file/compose-file-v3/#restart